File: source/code/php/authentication/authenticate_basic_req.php
$request = \Stormpath\Authc\Api\Request::createFromGlobals();
$result = (new \Stormpath\Authc\Api\BasicRequestAuthenticator($application))->authenticate($request);

$account = $result->getApiKey()->account;
$apiKey = $result->getApiKey();

File: source/code/java/authentication/authenticate_basic_req.java
byte[] authBytes = (apiKey.getId() + ":" + apiKey.getSecret()).getBytes();
String basic = "Basic " + DatatypeConverter.printBase64Binary(authBytes);
HttpRequest request = HttpRequests.method(HttpMethod.GET)
    .addHeader("Authorization", new String[] {basic})
    .build();

ApiAuthenticationResult result = Applications
    .apiRequestAuthenticator(application)
    .authenticate(request);

Account account = result.getAccount();
